lib/modules/datasource/rubygems/http.ts

```diff
@@ -0,0 +1,18 @@
+import { Http } from '../../../util/http';
+import { Throttle } from '../../../util/http/throttle';
+import { parseUrl } from '../../../util/url';
+
+export class RubygemsHttp extends Http {
+  protected override getThrottle(url: string): Throttle | null {
+    const host = parseUrl(url)?.host;
+
+    if (host === 'rubygems.org') {
+      // rubygems.org has a rate limit of 10 per second, so we use a more conservative 8
```

Comment:
Are you sure it's the 10/s which applies and not 15/s?

Reply:
But I'm ok with this actually, considering how many workers we may have in parallel

Reply:
According to [this page](https://blog.rubygems.org/2023/02/22/dependency-api-deprecation.html), dependencies API is the one which we use as fallback:

> The API has a single endpoint, `/api/v1/dependencies` (returning responses as Marshal or as JSON), accepting a single comma-separated gems query parameter.

It means, that other endpoints are in "API and website" category, which is 10 per second.